.organizationmap

```diff
@@ -27,6 +27,17 @@ Acaleph <contact@acale.ph> Alistair Israel <aisrael@gmail.com>
 Alcatel Lucent <contact@alcatel-lucent.com> Joseph McDonald <joseph.mcdonald@alcatel-lucent.com>
 Alcatel Lucent <contact@alcatel-lucent.com> Ker Liu <ker.liu@alcatel-lucent.com>
 Aliyun <contact@aliyun.com> Jeffrey Lu <lzhng2000@aliyun.com>
+Alibaba<contact@alibaba-inc.com> James Liu <james.liu@alibaba-inc.com>
+Alibaba<contact@alibaba-inc.com> Pan Liu <wanjun.lp@alibaba-inc.com>
```

Comment:
@jupiturliu i failed to find any of these contributors in the .mailmap or `git log`, are you sure they are the right list you want to add?